docs/en/04_Changelogs/beta/4.10.0-beta1.md

```diff
@@ -0,0 +1,55 @@
+# 4.10.0 beta1
+
+## Overview
+
+- [Stable module forcing installation of unstable framework](#unstable-framework)
+- [Dropping support for PHP 7.1 and PHP 7.2](#phpeol)
+- [Features and enhancements](#features-and-enhancements)
+- [Bugfixes](#bugfixes)
+
+## Stable module forcing installation of unstable framework{#unstable-framework}
+
+If you recently updated your project's `composer.json` file, you may have accidentally got an unstable version of `silverstripe/framework`.
+
+In the process of publishing Silverstripe CMS Recipe 4.10.0-beta1, we ended up releasing a few modules as "stable" but with a requirement on the unstable `silverstripe/framework` 4.10. For most projects, this won't be a ploblem if your `composer.json` file uses the `"prefer-stable": true` flag. However, if you explicitely require the very latest version of a module that depends on `silverstripe/framework ^4.10`, composer will install an unstable version of `silverstripe/framework`.
```

Comment:
```suggestion
In the process of publishing Silverstripe CMS Recipe 4.10.0-beta1, we ended up releasing a few modules as "stable" but with a requirement on the unreleased `silverstripe/framework` 4.10. For most projects, this won't be a ploblem if your `composer.json` file uses the `"prefer-stable": true` flag. However, if you explicitely require the very latest version of a module that requires `silverstripe/framework ^4.10` then composer may install an unstable version of `silverstripe/framework` e.g. `4.x-dev` or `4.10.0-beta`.
```

Reply:
I've switched "unstable" to "unreleased" and "will" to "may".

I don't think we need to give the examples because they are also below.

I would rather keep "depends on" because I'm already using "require in this sentence.